### PR TITLE
More ACU functions

### PR DESCRIPTION
**Pull request by joel16**
- ACU_GetProxy* functions
- ACU_GetErrorCode functions
- ACU_GetSSID
- Added an enumerator for security mode. 
- Minor consistency stuff.
- More explicit documentation.

### CODE REVIEW

File: libctru/include/3ds/services/ac.h [hunk 1]
```diff
@@ -14,25 +26,67 @@ void acExit(void);
 Result acWaitInternetConnection(void);
 
 /**
- * @brief Gets the current Wifi status.
- * @param out Pointer to output the current Wifi status to. (0 = not connected, 1 = O3DS Internet, 2 = N3DS Internet)
+ * @brief Gets the connected WiFi status.
```
Comment (fincs) [inline libctru/include/3ds/services/ac.h]: Keep Wifi spelled as Wifi.

File: libctru/include/3ds/services/ac.h [hunk 2]
```diff
@@ -14,25 +26,67 @@ void acExit(void);
...
+
+/**
+ * @brief Gets the connected network's proxy port.
+ * @param out Pointer to output the proxy port to.
+ */
+Result ACU_GetProxyPort(u32 *out);
+
+/**
+ * @brief Gets the connected network's proxy username.
+ * @param username Pointer to output the proxy username to.
```
Comment (fincs) [inline libctru/include/3ds/services/ac.h]: Buffer size should be documented.

**Comment by fincs** [comment]
Also please rebase on latest master to fix CI error.

**Comment by joel16** [comment]
Rebase seems to be mucking up for me, I'll just close the PR and make a new one.

**Pull request closed without merging**